features/json-modules.yml.dist

```diff
@@ -2,14 +2,30 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: false
+  baseline: low
+  baseline_low_date: 2025-04-29
```

Comment:
This looks right. A correction made in https://github.com/mdn/browser-compat-data/issues/26811